src/queue.js

```diff
@@ -0,0 +1,69 @@
+'use strict'
+
+class Node {
```

Comment:
in the future, you can define a ``` Node ``` in another file and import into whatever file you need to so you dont have duplicate code.